config/routes.rb

```diff
@@ -1,12 +1,16 @@
 
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
-
-  # You can have the root of your site routed with "root"
-  root 'cards#index'
+  resources :cards
+  resources :users, only: [:new, :create]
+  resources :sessions, only: [:new, :create, :destroy]
+
+  get '/sign_up', to: 'users#new', as: :sign_up
+  get '/log_in', to: 'sessions#new', as: :log_in
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.